tests/test_linear_model.py

```diff
@@ -2,19 +2,25 @@
 
 import numpy as np
 import unittest
-
-from prog_models.models.test_models.linear_models import FNoneNoEventStateLM
-from prog_models.models.test_models.linear_thrown_object import LinearThrownObject
+import copy
+import pickle
+from prog_models.models.test_models.linear_thrown_object import *
+from prog_models.models.test_models.linear_models import *
```

Comment:
Sounds good. 

Also with regards to test_composite.py, it was originally in test_base_models.py.

So, there were no real changes that were done to the tests, I just moved them to a different file since we were going to move them to a different file for testing anyways (I think we agreed on these changes, but I could be wrong. I can move the tests back and remove the file from this branch).

Reply:
Ah okay, that makes sense. I'll leave this one up to @teubert